accelerator/config.c

```diff
@@ -92,56 +98,59 @@ status_t cli_config_set(char* conf_file, ta_config_t* const info, iota_config_t*
     // File configuration
     case CONF_CLI: {
       size_t arg_len = strlen(value);
-      strncpy(conf_file, value, arg_len);
-      conf_file[arg_len] = '\0';
+      strncpy(ta_core->conf_file, value, arg_len);
+      ta_core->conf_file[arg_len] = '\0';
       break;
     }
   }
   return SC_OK;
 }
 
-status_t ta_config_default_init(ta_config_t* const info, iota_config_t* const iconf, ta_cache_t* const cache,
-                                iota_client_service_t* const service) {
+status_t ta_config_default_init(ta_core_t* ta_core) {
```

Comment:
You did attempt to refactor ta_config, that is expected to appear in another pull request. Can you send another pull request which emphasizes on ta_config refactoring instead?

Reply:
Again, always minimize the functional changes.